rails/config/routes.rb

```diff
@@ -547,8 +545,6 @@
 
     get '/resources/:type/:id_or_filter_value(/:slug)' => 'home#stem_resources', :as => :redirect_stem_resources
 
-    get '/:controller(/:action(/:id))'
```

Comment:
Aha -- maybe this change is the one that caused the 122 new routing errors! 

Here are the routes that are now not matching:

```
   1 Failure/Error: %li= link_to 'edit settings', url_for(action: :edit, id:admin_settings.id) ActionView::Template::Error: No route matches {:action=>"edit", :controller=>"admin/settings", :id=>nil}
   1 Failure/Error: delete :destroy, id: mock_content_id ActionController::UrlGenerationError: No route matches {:action=>"destroy", :controller=>"admin/commons_licenses", :id=>"1056"}
   1 Failure/Error: delete :destroy, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :controller=>"api/api"}
   1 Failure/Error: delete :destroy, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :controller=>"portal/learners"}
   1 Failure/Error: delete :destroy, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :controller=>"portal/nces06_districts"}
   1 Failure/Error: delete :destroy, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :controller=>"portal/nces06_schools"}
   1 Failure/Error: delete :destroy, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :controller=>"saveable/sparks/measuring_resistances"}
   1 Failure/Error: expect(get :edit).to render_template('edit') ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"admin/project_links"}
   1 Failure/Error: get :archive, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"archive", :controller=>"external_activities"}
   1 Failure/Error: get :authoring_site_redirect, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"authoring_site_redirect", :controller=>"home"}
   1 Failure/Error: get :authorize_show, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"authorize_show", :controller=>"portal/learners"}
   1 Failure/Error: get :bundle_report, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"bundle_report", :controller=>"portal/learners"}
   1 Failure/Error: get :copy, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"copy", :controller=>"external_activities"}
   1 Failure/Error: get :create_by_learner, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"create_by_learner", :controller=>"dataservice/bucket_log_items_metal"}
   1 Failure/Error: get :create_by_name, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"create_by_name", :controller=>"dataservice/bucket_log_items_metal"}
   1 Failure/Error: get :create_by_protocol_version, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"create_by_protocol_version", :controller=>"dataservice/external_activity_data"}
   1 Failure/Error: get :current_clazz, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"current_clazz", :controller=>"portal/clazzes"}
   1 Failure/Error: get :current_clazz, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"current_clazz", :controller=>"portal/learners"}
   1 Failure/Error: get :description, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"description", :controller=>"portal/nces06_schools"}
   1 Failure/Error: get :edit ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"admin/project_links"}
   1 Failure/Error: get :edit ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"security_questions"}
   1 Failure/Error: get :edit, id: mock_content_id ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"admin/commons_licenses", :id=>"1054"}
   1 Failure/Error: get :edit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"portal/learners"}
   1 Failure/Error: get :edit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"portal/nces06_districts"}
   1 Failure/Error: get :edit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"portal/nces06_schools"}
   1 Failure/Error: get :edit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"portal/student_clazzes"}
   1 Failure/Error: get :edit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"saveable/sparks/measuring_resistances"}
   1 Failure/Error: get :external_report, id: FactoryBot.create(:portal_offering).to_param ActionController::UrlGenerationError: No route matches {:action=>"external_report", :controller=>"portal/offerings", :id=>"167"}
   1 Failure/Error: get :failed_creation, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"failed_creation", :controller=>"portal/teachers"}
   1 Failure/Error: get :failure, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"failure", :controller=>"auth"}
   1 Failure/Error: get :handle_jnlp_session, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"handle_jnlp_session", :controller=>"portal/learners"}
   1 Failure/Error: get :index, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"index", :controller=>"api/api"}
   1 Failure/Error: get :index, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"index", :controller=>"portal/bookmarks"}
   1 Failure/Error: get :isalive, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"isalive", :controller=>"auth"}
   1 Failure/Error: get :launch_status, :id => learner.offering.id ActionController::UrlGenerationError: No route matches {:action=>"launch_status", :controller=>"portal/offerings_metal", :id=>"169"}
   1 Failure/Error: get :launch_status, :id => offering.id ActionController::UrlGenerationError: No route matches {:action=>"launch_status", :controller=>"portal/offerings_metal", :id=>"168"}
   1 Failure/Error: get :log_links, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"log_links", :controller=>"api/v1/classes"}
   1 Failure/Error: get :matedit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"matedit", :controller=>"external_activities"}
   1 Failure/Error: get :missing_installer, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"missing_installer", :controller=>"home"}
   1 Failure/Error: get :preflight ActionController::UrlGenerationError: No route matches {:action=>"preflight", :controller=>"misc"}
   1 Failure/Error: get :questions, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"questions", :controller=>"passwords"}
   1 Failure/Error: get :registration_successful, {:type => 'student'} ActionController::UrlGenerationError: No route matches {:action=>"registration_successful", :controller=>"users", :type=>"student"}
   1 Failure/Error: get :registration_successful, {:type => 'teacher'} ActionController::UrlGenerationError: No route matches {:action=>"registration_successful", :controller=>"users", :type=>"teacher"}
   1 Failure/Error: get :republish, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"republish", :controller=>"external_activities"}
   1 Failure/Error: get :reset, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"reset", :controller=>"passwords"}
   1 Failure/Error: get :set_private_before_matedit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"set_private_before_matedit", :controller=>"external_activities"}
   1 Failure/Error: get :show, id: 'a' * 36 ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"external_activities", :id=>"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"} missing required keys: [:id]
   1 Failure/Error: get :show, id: mock_content_id ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"admin/commons_licenses", :id=>"1053"}
   1 Failure/Error: get :show, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"api/api"}
   1 Failure/Error: get :show, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"dataservice/bucket_loggers"}
   1 Failure/Error: get :show, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"portal/learners"}
   1 Failure/Error: get :show, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"portal/nces06_schools"}
   1 Failure/Error: get :show, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"portal/student_clazzes"}
   1 Failure/Error: get :show, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show", :controller=>"saveable/sparks/measuring_resistances"}
   1 Failure/Error: get :show_by_learner, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show_by_learner", :controller=>"dataservice/bucket_loggers"}
   1 Failure/Error: get :show_by_name, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show_by_name", :controller=>"dataservice/bucket_loggers"}
   1 Failure/Error: get :show_log_items_by_learner, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show_log_items_by_learner", :controller=>"dataservice/bucket_loggers"}
   1 Failure/Error: get :show_log_items_by_name, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"show_log_items_by_name", :controller=>"dataservice/bucket_loggers"}
   1 Failure/Error: get :successful_creation, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"successful_creation", :controller=>"portal/teachers"}
   1 Failure/Error: get :teacher_admin_or_manager, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"teacher_admin_or_manager", :controller=>"portal/teachers"}
   1 Failure/Error: get :unarchive, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"unarchive", :controller=>"external_activities"}
   1 Failure/Error: get :update_forms, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update_forms", :controller=>"admin/permission_forms"}
   1 Failure/Error: get :update_learners, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update_learners", :controller=>"report/learner"}
   1 Failure/Error: get :verify_logged_in, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"verify_logged_in", :controller=>"auth"}
   1 Failure/Error: get :visit, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"visit", :controller=>"portal/bookmarks"}
   1 Failure/Error: post :create, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"create", :controller=>"api/api"}
   1 Failure/Error: post :create, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"create", :controller=>"dataservice/external_activity_data"}
   1 Failure/Error: post :create_by_email, @params ActionController::UrlGenerationError: No route matches {:action=>"create_by_email", :controller=>"passwords", :password=>{:email=>"test@test.com"}}
   1 Failure/Error: post :remove_material, {} ActionController::UrlGenerationError: No route matches {:action=>"remove_material", :controller=>"api/v1/materials_collections"}
   1 Failure/Error: post :sort, {ids: [0]} ActionController::UrlGenerationError: No route matches {:action=>"sort", :controller=>"api/v1/teacher_classes", :ids=>["0"]}
   1 Failure/Error: post :sort, {ids: [other_teacher_clazz.id]} ActionController::UrlGenerationError: No route matches {:action=>"sort", :controller=>"api/v1/teacher_classes", :ids=>["348"]}
   1 Failure/Error: post :sort, {ids: [teacher_clazz3.id, teacher_clazz.id, teacher_clazz2.id]} ActionController::UrlGenerationError: No route matches {:action=>"sort", :controller=>"api/v1/teacher_classes", :ids=>["351", "353", "355"]}
   1 Failure/Error: post :sort, {} ActionController::UrlGenerationError: No route matches {:action=>"sort", :controller=>"api/v1/teacher_classes"}
   1 Failure/Error: post :sort_materials, {} ActionController::UrlGenerationError: No route matches {:action=>"sort_materials", :controller=>"api/v1/materials_collections"}
   1 Failure/Error: post :update, params ActionController::UrlGenerationError: No route matches {:action=>"update", :admin_cohort=>{:project_id=>"49", :name=>nil}, :controller=>"admin/cohorts"}
   1 Failure/Error: post :update, params ActionController::UrlGenerationError: No route matches {:action=>"update", :admin_cohort=>{:project_id=>"52", :name=>nil}, :controller=>"admin/cohorts"}
   1 Failure/Error: post :update, params ActionController::UrlGenerationError: No route matches {:action=>"update", :admin_cohort=>{:project_id=>"55", :name=>nil}, :controller=>"admin/cohorts"}
   1 Failure/Error: post :update, params ActionController::UrlGenerationError: No route matches {:action=>"update", :admin_cohort=>{:project_id=>"58", :name=>nil}, :controller=>"admin/cohorts"}
   1 Failure/Error: post method, {clazz_id: clazz1.id + 1} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :clazz_id=>"33", :controller=>"api/v1/bookmarks"}
   1 Failure/Error: post method, {clazz_id: clazz1.id + 1} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :clazz_id=>"45", :controller=>"api/v1/bookmarks"}
   1 Failure/Error: post method, {clazz_id: clazz1.id + 1} ActionController::UrlGenerationError: No route matches {:action=>"update", :clazz_id=>"29", :controller=>"api/v1/bookmarks"}
   1 Failure/Error: post method, {clazz_id: clazz1.id + 1} ActionController::UrlGenerationError: No route matches {:action=>"update", :clazz_id=>"43", :controller=>"api/v1/bookmarks"}
   1 Failure/Error: post method, {clazz_id: clazz1.id} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :clazz_id=>"34", :controller=>"api/v1/bookmarks"}
   1 Failure/Error: post method, {clazz_id: clazz1.id} ActionController::UrlGenerationError: No route matches {:action=>"update", :clazz_id=>"30", :controller=>"api/v1/bookmarks"}
   1 Failure/Error: post method, {} ActionController::UrlGenerationError: No route matches {:action=>"remove_material", :controller=>"api/v1/materials_collections"}
   1 Failure/Error: post method, {} ActionController::UrlGenerationError: No route matches {:action=>"sort_materials", :controller=>"api/v1/materials_collections"}
   1 Failure/Error: put :update, :id => mock_content_id, params_key => {:params => 'params'} ActionController::UrlGenerationError: No route matches {:action=>"update", :commons_license=>{:params=>"params"}, :controller=>"admin/commons_licenses", :id=>"1055"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :activity_feedback=>{:learner_id=>"79", :activity_feedback_id=>"13", :score=>"10", :rubric_feedback=>{:C1=>{:id=>"R1", :score=>"1", :label=>"Beginning", :description=>"Not meeting expected goals."}, :C2=>{:id=>"R2", :score=>"2", :label=>"Developing", :description=>"Approaching proficiency."}}, :text_feedback=>"good work"}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :activity_feedback_opts_v2=>{:enable_text_feedback=>false, :score_type=>"auto", :max_score=>"10", :use_rubric=>false, :activity_index=>"0"}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :activity_feedback_opts_v2=>{:enable_text_feedback=>false, :score_type=>"none", :max_score=>"10", :use_rubric=>true, :activity_index=>"0"}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :activity_feedback_opts_v2=>{:enable_text_feedback=>false, :score_type=>"none", :max_score=>"20", :use_rubric=>false, :activity_index=>"0"}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :activity_feedback_v2=>{:score=>"10", :rubric_feedback=>{:C1=>{:id=>"R1", :score=>"1", :label=>"Beginning", :description=>"Not meeting expected goals."}, :C2=>{:id=>"R2", :score=>"2", :label=>"Developing", :description=>"Approaching proficiency."}}, :text_feedback=>"good work", :student_user_id=>"942", :activity_index=>"0"}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :actvity_feedback_opts=>{:activity_feedback_id=>"11", :enable_text_feedback=>false, :score_type=>"none", :max_score=>"10", :use_rubric=>true}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :actvity_feedback_opts=>{:activity_feedback_id=>"7", :enable_text_feedback=>false, :score_type=>"auto", :max_score=>"10", :use_rubric=>false}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :actvity_feedback_opts=>{:activity_feedback_id=>"9", :enable_text_feedback=>false, :score_type=>"none", :max_score=>"20", :use_rubric=>false}, :controller=>"api/v1/reports"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"api/api"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"external_activities"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"passwords"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/courses"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/learners"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/nces06_districts"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/nces06_schools"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/offerings"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/school_memberships"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/student_clazzes"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"portal/subjects"}
   1 Failure/Error: put :update, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"saveable/sparks/measuring_resistances"}
   1 Failure/Error: put :update_collections, post_params ActionController::UrlGenerationError: No route matches {:action=>"update_collections", :controller=>"external_activities", :materials_collection_id=>["8"]}
   1 Failure/Error: put :update_collections, post_params ActionController::UrlGenerationError: No route matches {:action=>"update_collections", :controller=>"external_activities", :materials_collection_id=>[]}
   1 Failure/Error: xhr :get, :stats, {}, {} ActionController::UrlGenerationError: No route matches {:action=>"stats", :controller=>"misc"}
   1 Failure/Error: xhr :post, :sort_offerings, @params ActionController::UrlGenerationError: No route matches {:action=>"sort_offerings", :clazz_offerings=>["143", "144", "145", "146"], :controller=>"portal/clazzes"}
   2 Failure/Error: expect(get :edit).to render_template('edit') ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"admin/cohorts"}
   2 Failure/Error: post action, {} ActionController::UrlGenerationError: No route matches {:action=>"copy", :controller=>"api/v1/teacher_classes"}
   2 Failure/Error: post action, {} ActionController::UrlGenerationError: No route matches {:action=>"set_active", :controller=>"api/v1/teacher_classes"}
   2 Failure/Error: post action, {} ActionController::UrlGenerationError: No route matches {:action=>"sort", :controller=>"api/v1/teacher_classes"}
   2 Failure/Error: post method, {} ActionController::UrlGenerationError: No route matches {:action=>"destroy", :controller=>"api/v1/bookmarks"}
   2 Failure/Error: post method, {} ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"api/v1/bookmarks"}
   2 Failure/Error: put :update, opts ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"api/v1/reports", :rubric_v2=>{:rubric=>{"test_rubric"=>"content"}}}
   3 Failure/Error: get :edit ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"admin/cohorts"}
   3 Failure/Error: get :index ActionController::UrlGenerationError: No route matches {:action=>"index", :controller=>"help"}
   3 Failure/Error: put :update, @params_for_update ActionController::UrlGenerationError: No route matches {:action=>"update", :controller=>"security_questions", :security_questions=>{:question0=>{:question_idx=>"0", :answer=>"test answer"}, :question1=>{:question_idx=>"1", :answer=>"test answer"}, :question2=>{:question_idx=>"2", :answer=>"test answer"}}}
   4 Failure/Error: post :create_by_login, @params ActionController::UrlGenerationError: No route matches {:action=>"create_by_login", :controller=>"passwords", :login=>"forgetful_jones"}

```